source/prerequisites.rst

```diff
@@ -77,6 +77,30 @@ Here is a configuration example for ``Nginx`` web server using ``php-fpm``.
         }
     }
 
+IIS configuration
+^^^^^^^^^^^^^^^^^
+
+Here is a configuration exemple for Microsoft IIS.
+
+.. code-block:: nginx
+
+   <?xml version="1.0" encoding="UTF-8"?>
+   <configuration>
+       <system.webServer>
+           <rewrite>
+               <rules>
+                   <rule name="Rewrite to index.php">
+                       <match url="^(.*)$" />
+                       <conditions>
+                           <add input="{REQUEST_FILENAME}" matchType="IsFile" negate="true" />
```

Comment:
Case sensitivity could be forced here, to ensure that it will act the same as on other operating systems.
```suggestion
                           <add input="{REQUEST_FILENAME}" matchType="IsFile" ignoreCase="false" negate="true" />
```